polartools/diffraction.py

```diff
@@ -467,7 +469,7 @@ def load_series(
                     scan,
```

Comment:
Do you need to move the `kwargs_scan = kwargs.copy()` in line 459 to be inside this `for` loop?